src/activation.js

```diff
@@ -49,6 +49,8 @@ function processDeactivatable(navigationInstruction: NavigationInstruction, call
       }
     }
 
+    navigationInstruction.router.couldDeactivate = true;
```

Comment:
This will treat `canDeactivate` and `deactivate` the same way. Does that have any implications ?

Reply:
No. I originally had a check for `canDeactivate` but it served no purpose and because of the naming of the variable `ignoreResult` I found that the coded was less readable. 

```
if (!ignoreResult) {
  navigationInstruction.router.couldDeactivate = true;
}
```

That's just gunna cause someone to waste 30 minutes figuring out what `ignoreResult` is and why it has anything to do with `couldDeactivate`.